udata_front/theme/gouvfr/assets/less/content/links.less

```diff
@@ -82,6 +82,10 @@ a {
     display: inline;
 }
 
+.text-color-blue-link {
```

Comment:
There is an existing `text-blue-400` to do this

Reply:
Done in a45f096f